neuralmonkey/util/word2vec.py

```diff
@@ -28,12 +36,11 @@ def __init__(self, path: str, encoding: str = "utf-8") -> None:
             header = next(f_data)
             emb_size = int(header.split()[1])
 
-            # Add zero embeddings for padding, start, and end token
+            # Add zero embeddings for padding, start, end, and unk token
+            embedding_vectors.append(np.zeros(emb_size))
             embedding_vectors.append(np.zeros(emb_size))
             embedding_vectors.append(np.zeros(emb_size))
             embedding_vectors.append(np.zeros(emb_size))
```

Comment:
Ale no tak. Vkladame zero embeddings pro vsechny SPECIAL_TOKENS, tak to hod do smycky, resp. to udelej zavisle na tom poli SPECIAL_TOKENS :)
(co kdyz se ta mnozine v budoucnu zvetsi/zmensi)

Reply:
Šikana!

Reply:
Ale zase budu víc commitů ve statistice ;-)